spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/dsl/MongoDbOutboundGatewaySpec.java

```diff
@@ -87,4 +88,9 @@ public MongoDbOutboundGatewaySpec collectionNameExpression(String collectionName
 		return this;
 	}
 
+	public <P> MongoDbOutboundGatewaySpec collectionCallback(CollectionCallback<P> collectionCallback) {
```

Comment:
Missed fix.
Actually it would be great to have JavaDocs on all public API of this class.
Since we are now in Java configuration era that really is very useful to get a hint about method just during typing 😄